packages/services/api/src/modules/auth/lib/authz.ts

```diff
@@ -366,8 +367,8 @@ const permissionsByLevel = {
     z.literal('laboratory:describe'),
     z.literal('laboratory:modify'),
     z.literal('laboratory:modifyPreflightScript'),
-    z.literal('schema:loadFromRegistry'),
     z.literal('schema:compose'),
+    z.literal('usage:report'),
```

Comment:
TODO: remove for now; will introduce it when we implement organization access token support in usage reporting service